### ISSUE TITLE
[BUG] - resetOnClose option doesn't work in navbar search plugin

### ISSUE THREAD

**Issue by toxpal** [ColorlibHQ/AdminLTE#5162]
Navbar search plugin documentation says that input should not be reset after closing/hiding search field bny default - https://adminlte.io/docs/3.2/javascript/navbar-search.html

However, this is not true - once search field is closed by clicking X button, input content (search keyword) is cleared too, and manually setting **resetOnClose** option to **false** doesn't help.

The same bug exists in online demo - https://adminlte.io/themes/v3/index2.html - type anything into search field, click the right X button to hide search field and input will be reset immediately.

I think it happens because plugin is not properly integrated into AdminLTE, or it's broken. I belioeve so because if I try to manually set an option as per documentation, JavaScript console in browser debugging tools displays error:
`SiteSearch is not a function`

**Comment by ajiho** [comment]
Yes, I noticed that the description in the source code and documentation is inconsistent

**Comment by puikinsh** [comment]
Closing as part of issue triage. This issue is from v3/early v4 development and is either resolved, outdated, or outside the current scope. If the issue persists with v4.0.0-rc7, please open a new issue.
